backend/inspirehep/search/aggregations.py

```diff
@@ -202,3 +202,34 @@ def hep_collection_aggregation(order, title="Collection", agg_type="checkbox"):
             "meta": {"title": title, "order": order, "type": agg_type},
         }
     }
+
+
+def hep_rpp(order, title="Exclude RPP", agg_type="checkbox"):
+    return {
+        "rpp": {
+            "filters": {
```

Comment:
wouldn't it be easier to use `filter` given you've only one filter? in that way things would be less nested too I think.